src/main/java/com/google/cloud/spanner/hibernate/SpannerTableExporter.java

```diff
@@ -41,17 +45,36 @@ public SpannerTableExporter(SpannerDialect spannerDialect) {
   }
 
   @Override
-  public String[] getSqlCreateStrings(Table exportable, Metadata metadata) {
-    return new String[]{"test_placeholder"};
+  public String[] getSqlCreateStrings(Table table, Metadata metadata) {
+    /* The current implementation does not support interleaved tables for collections
+     * or UNIQUE constraints/indexes for relationships
+     * */
+    String createTableTemplate = this.spannerDialect.getCreateTableString()
+        + " {0} ({1}) PRIMARY KEY ({2})";
+
+    StringJoiner primaryKeyColNames = new StringJoiner(",");
+
+    table.getPrimaryKey().getColumns().forEach(col -> primaryKeyColNames.add(col.getName()));
+
+    StringJoiner colsAndTypes = new StringJoiner(",");
+
+    ((Iterator<Column>) table.getColumnIterator()).forEachRemaining(col -> colsAndTypes
+        .add(col.getName() + " " + col.getSqlType(this.spannerDialect, metadata) + (col.isNullable()
+            ? this.spannerDialect.getNullColumnString() : " not null")));
```

Comment:
I think you can extract this to a private helper method to make more readable and clarify purpose of the String concatentations.

i.e.
```
private String createColumnTypeFragment(Column column) {
  return column.getName() + " " + column.getSqlType(...) + ...;
}
```

Reply:
I generally agree with extracting little methods like this but currently I don't think that's a good idea in this specific situation. 

Because this snippet of code is not used anywhere else I think a reader basically gets disturbed in his reading having to shift his eyes half way down the page to continue reading and then having to find his place half way back up the page again to continue reading while we don't gain any benefit of reducing code repetition.

Reply:
Sounds good.